src/applications/claims-status/tests/components/appeals-v2/AppealInfo.unit.spec.jsx

```diff
@@ -42,13 +46,15 @@ describe('<AppealInfo>', () => {
 
   it('should render its children', () => {
     const screen = render(
-      <MemoryRouter>
-        <Routes>
-          <Route element={<AppealInfo {...defaultProps} />}>
-            <Route index element={<TestComponent />} />
-          </Route>
-        </Routes>
-      </MemoryRouter>,
+      <Provider store={store}>
```

Comment:
What is the purpose of this modification?

Reply:
We are testing the version of the `AppealInfo` component that doesn't use Redux here so I'm not sure what the purpose of adding a `Provider` is